src/api/IBMiContent.ts

```diff
@@ -360,16 +404,10 @@ export default class IBMiContent {
       `;
       results = await this.runSQL(statement);
     } else {
-      await this.ibmi.runCommand({
-        command: `DSPOBJD OBJ(QSYS/*ALL) OBJTYPE(*LIB) DETAIL(*TEXTATR) OUTPUT(*OUTFILE) OUTFILE(${tempLib}/${TempName})`,
-        noLibList: true
-      });
-      results = await this.getTable(tempLib, TempName, TempName, true);
-
+      results = await this.getQTempTable([`CALL QSYS2.QCMDEXC('DSPOBJD OBJ(QSYS/*ALL) OBJTYPE(*LIB) DETAIL(*TEXTATR) OUTPUT(*OUTFILE) OUTFILE(QTEMP/LIBLIST)`], "LIBLIST");
```

Comment:
I think your command is missing a quote and parenthesis:

```suggestion
      results = await this.getQTempTable([`CALL QSYS2.QCMDEXC('DSPOBJD OBJ(QSYS/*ALL) OBJTYPE(*LIB) DETAIL(*TEXTATR) OUTPUT(*OUTFILE) OUTFILE(QTEMP/LIBLIST)')`], "LIBLIST");
```

Reply:
The command is executed when SQL is not enabled and library descriptions are enabled, and results in this error and an empty library list view:

![image](https://github.com/codefori/vscode-ibmi/assets/13275072/e934901e-96f9-4f12-bfe2-c34aa5140bf2)

Reply:
Fixed in commit [8887227](https://github.com/codefori/vscode-ibmi/pull/1780/commits/8887227ca3bb471be72cacbd33a31a148cc08ebf)